### ISSUE TITLE
[bug] I18n with Lingui

### ISSUE THREAD

**Issue by sasicodes** [rainbow-me/rainbowkit#1545]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### RainbowKit Version

1.1.0

### wagmi Version

1.4.3

### Current Behavior

<img width="1275" alt="image" src="https://github.com/rainbow-me/rainbowkit/assets/29498872/338815bf-e182-4058-8db6-5873811aab27">

**Comment by saltyfacu** [comment]
Same with wagmi 1.3.11

**Comment by manzgoeggel** [comment]
Same here. 

`
  "next": "^13.5.4",
   "viem": "^1.15.4",
   "wagmi": "^1.4.3"
`
# next.config.js:

```
/** @type {import('next').NextConfig} */
const nextConfig = {
	reactStrictMode: false,
	webpack: (config) => {
		config.resolve.fallback = { fs: false, net: false, tls: false };
		return config;
	},
	i18n: {
		locales: ["en"],

		defaultLocale: "en",
	},
};

module.exports = nextConfig;




```

**Comment by reyser-eth** [comment]
same here
event the example for Minting Next Js is have issue I18n 
"@rainbow-me/rainbowkit": "^1.1.0",
"viem": "^1.15.4",
"wagmi": "^1.4.3",

**Comment by vienalpha** [comment]
same here
event the example for Minting Next Js is have issue I18n

**Comment by impolska742** [comment]
same here! 
`

Collecting page data ...
--
17:31:03.222 | file:///vercel/path0/node_modules/@rainbow-me/rainbowkit/dist/chunk-DPMVRV6C.js:514
17:31:03.222 | import { I18n } from "i18n-js";
17:31:03.222 | ^^^^
17:31:03.222 | SyntaxError: Named export 'I18n' not found. The requested module 'i18n-js' is a CommonJS module, which may not support all module.exports as named exports.
17:31:03.222 | CommonJS modules can always be imported via the default export, for example using:
17:31:03.222 |  
17:31:03.222 | import pkg from 'i18n-js';
17:31:03.222 | const { I18n } = pkg;
17:31:03.222 |  
17:31:03.222 | at ModuleJob._instantiate (node:internal/modules/esm/module_job:124:21)
17:31:03.222 | at async ModuleJob.run (node:internal/modules/esm/module_job:190:5)
17:31:03.225 |  
17:31:03.225 | > Build error occurred
17:31:03.242 | Error: Failed to collect page data for /analytics
17:31:03.243 | at /vercel/path0/node_modules/next/dist/build/utils.js:1178:15 {
17:31:03.244 | type: 'Error'
17:31:03.244 | }
17:31:03.375 | error Command failed with exit code 1.
17:31:03.376 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
17:31:03.400 | Error: Command "yarn run build" exited with 1

`

**Comment by louis-md** [comment]
Same! Had to downgrade to 1.0.11

**Comment by manzgoeggel** [comment]
The examples in this root repo are even failing lmao

**Comment by herbig** [comment]
++

**Comment by sasicodes** [comment]
Downgrading to `1.0.12` works for now.

**Comment by DanielSinclair** [comment]
Thanks for reporting; investigating. Can anyone share a minimal reproducible example and provide more information about your Next.js version and project configuration?

**Comment by jvaleskadevs** [comment]
> Thanks for reporting; investigating. Can anyone share a minimal reproducible example and provide more information about your Node.js version and project configuration?

Sure. Here we go.

CLI:
```yarn create @rainbow-me/rainbowkit```
```yarn dev```

Node version: `v16.17.1`

Error:
```
file:///path_to_my_project/node_modules/@rainbow-me/rainbowkit/dist/chunk-DPMVRV6C.js:514
import { I18n } from "i18n-js";
         ^^^^
SyntaxError: Named export 'I18n' not found. The requested module 'i18n-js' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'i18n-js';
const { I18n } = pkg;

    at ModuleJob._instantiate (node:internal/modules/esm/module_job:123:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:189:5)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:533:24)
    at async importModuleDynamicallyWrapper (node:internal/vm/module:438:15) {
  page: '/'
}
```
Package.json:
```
{
  "name": "project_name",
  "private": true,
  "version": "0.1.0",
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint"
  },
  "dependencies": {
    "@rainbow-me/rainbowkit": "^1.1.0",
    "next": "^13.4.19",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "viem": "~1.12.2",
    "wagmi": "~1.4.2"
  },
  "devDependencies": {
    "@types/node": "^18.16.12",
    "@types/react": "^18.2.21",
    "eslint": "^8.15.0",
    "eslint-config-next": "^13.4.19",
    "typescript": "^5.0.4"
  }
}
```

Do you need anything more? Let me know and I will share it.

thanks.

**Comment by DanielSinclair** [comment]
This is now fixed in [v1.1.1](https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Frainbowkit%401.1.1). Apologies for the mishap; the downstream resolution problem in ESModule projects oddly did not appear in our `pnpm` monorepo.
